server/src/pages/homepage/view.js

```diff
@@ -9,14 +9,14 @@ class Head extends React.Component {
       <reactruntime.HeadTemplate {...this.props}>
         <script src={this.props.staticLink("/static/js/homepage-bundle.js")} async></script>
         <meta name="viewport" content="width=320, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
-        <meta name="description" content="Share anything on the web with anyone using Firefox Screenshots." />
+        <meta name="description" content="Share anything on the web with anyone using Firefox Screenshots." /> <!-- todo l10n: homePageDescription -->
```

Comment:
@stasm The fluent/fluent-react code seems like it's still evolving pretty rapidly. Is there a recommended, more stable library we could use instead, until the fluent APIs have some time to bake?